changelog_unreleased/html/pr-7273.md

```diff
@@ -0,0 +1,29 @@
+#### Fix HTML elements inside unknown namespace element ([#7273](https://github.com/prettier/prettier/pull/7273) by [@fisker](https://github.com/fisker))
```

Comment:
In HTML5, colons don't have special meaning in tag names, so mentioning "namespaces" is the changelog is incorrect. If not a bug, they're just an internal thing of the Angular parser.